container/openejb-core/src/test/java/org/apache/openejb/activemq/JMS2AMQTest.java

```diff
@@ -93,6 +93,8 @@ public Properties config() {
 
                 .p("mdbs", "new://Container?type=MESSAGE")
                 .p("mdbs.ResourceAdapter", "amq")
+                .p("mdbs.InstanceLimit", "30")
```

Comment:
maybe put a comment why it is needed because there is no high concurrency in this test (the 50 messages are sequential)

Two things which can go wrong in such a test are the sleep() - but normally it is also guarded by latches - and the implicit request scope which would stack messages if implicitly used but increasing instances and sessions is surprising upfront so a comment is likely worth it.